single-line-diagram-core/src/main/resources/defaultConstantColors.css

```diff
@@ -1,9 +1,26 @@
-svg {stroke: blue}
-.sld-node {stroke: none; fill: black}
-.sld-node-infos {stroke: none; fill: var(--sld-vl-color)}
+/* ----------------------------------------------------------------------- */
+/* File: defaultConstantColors.css --------------------------------------- */
+/* Stroke none & fill: black */
 .sld-label {stroke: none; fill: black}
-.sld-bus-connection {stroke: black}
+.sld-node {stroke: none; fill: black}
+.sld-flash {stroke: none; fill: black}
+.sld-lock {stroke: none; fill: black}
+.sld-graph-label {stroke: none; fill: black}
+/* Stroke black */
 .sld-disconnector {stroke: black}
-.sld-graph-label {stroke: black}
-.sld-wire {stroke: var(--sld-vl-color, #c80000)}
-.sld-busbar-section {stroke: var(--sld-vl-color, black)}
+.sld-busbar-section {stroke: black}
+.sld-bus-connection {stroke: black}
+/* Stroke red & fill: none */
+.sld-wire {stroke: #c80000}
+/* Stroke blue */
+.sld-load-break-switch {stroke: blue}
+.sld-load {stroke: blue}
+.sld-generator {stroke: blue}
+.sld-two-wt {stroke: blue}
+.sld-three-wt {stroke: blue}
+.sld-winding {stroke: blue}
+.sld-capacitor {stroke: blue}
+.sld-inductor {stroke: blue}
+.sld-pst {stroke: blue}
+.sld-svc {stroke: blue}
+.sld-vsc {stroke: blue}
```

Comment:
As said before, I think all style related to components should be in components.css. I think this file can be merged with baseVoltageConstantColors.css, as at the end there won't be anything specific with base voltage there. In fact both can maybe be deleted, as only labels are not components but their style is linked to components (and there's already an entry for labels in components.css). What do you think?